<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="LAPTOP-STP14Q1E" failures="1" tests="5" name="Personal Loan" time="0.077" errors="0" timestamp="2024-08-20T18:46:21 IST">
  <testcase classname="test.Day1" name="Test1" time="0.017">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.Day1.Test1(Day1.java:32)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1570)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- Test1 -->
  <testcase classname="test.Day2" name="personalLoan" time="0.002"/>
  <testcase classname="test.Day4" name="LoginAPIHomeLoan" time="0.002"/>
  <testcase classname="test.Day4" name="mobileLoginHomeLoan" time="0.002"/>
  <testcase classname="test.Day4" name="webloginHomeLoan" time="0.001"/>
  <testcase classname="test.Day1" name="Demo" time="0.0">
    <ignored/>
  </testcase> <!-- Demo -->
</testsuite> <!-- Personal Loan -->
